### PR TITLE
Regroup equipmentDeletion notifications

### PR DESCRIPTION
**Pull request by ayolab**
Signed-off-by: Ayoub LABIDI <ayoub.labidi@protonmail.com>

### CODE REVIEW

**Review by laurgarn — APPROVED** [review]

**Comment by laurgarn** [comment]
(Tested OK)

**Review by archendel — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]

File: src/main/java/org/gridsuite/study/notification/server/payload/NetworkImpactsNotificationPayload.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+package org.gridsuite.study.notification.server.payload;
+
+import java.util.Set;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class NetworkImpactsNotificationPayload {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/study/notification/server/payload/NetworkImpactsNotificationPayload.java:17]: Rename by NetworkImpactsInfos

File: src/main/java/org/gridsuite/study/notification/server/payload/EquipmentDeletionNotification.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ */
+
+package org.gridsuite.study.notification.server.payload;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class EquipmentDeletionNotification {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/study/notification/server/payload/EquipmentDeletionNotification.java:15]: Rename by EquipmentDeletionInfos

File: src/main/java/org/gridsuite/study/notification/server/payload/EquipmentDeletionNotification.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.study.notification.server.payload;
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/study/notification/server/payload/EquipmentDeletionNotification.java:8]: Rename by `dto`

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**